docs/ipdk-tofino.md

```diff
@@ -0,0 +1,116 @@
+Build IPDK on tofino
+====================
+
+# Overview
+This document describes how to build and run IPDK on Tofino HW. Similar steps apply when run on Tofino simulation model. Please refer to Tofino specific documentation for more details.
```

Comment:
Can we add a link to the document you are referring to here?

Reply:
I believe the documentation is proprietary.